src/gt4py/_core/definitions.py

```diff
@@ -491,22 +499,28 @@ def __rtruediv__(self, other: Any) -> NDArrayObject:
     def __pow__(self, other: NDArrayObject | Scalar) -> NDArrayObject:
         ...
 
-    def __eq__(self, other: NDArrayObject | Scalar) -> NDArrayObject:  # type: ignore[override] # mypy want to return `bool`
+    # mypy wants to return `bool`
```

Comment:
why this change? the comment explains the type ignore, therefore I prefer the old location.

Reply:
The automatic formatting broke it. If pre-commit black is fine with the old formatting then I will have to investigate why my ruff-lsp differs from it.